### PR TITLE
Specify 3.2 nightly

### PR DESCRIPTION
**Pull request by mcalmer**
We are short before branching and we need to run cucumber tests for 3.2 on nightly build.

### CODE REVIEW

**Comment by mcalmer** [comment]
@moio I have some trouble to get the server run. As they are in US, download.suse.de seems to point to a local mirror which has invalid data sometimes. Repos do not refresh correctly, because the signature file is still old or repomd.xml contains outdated data.

So I changed download.suse.de to download.nue.suse.com which provide correct data, but it means we access not local resources.

I think we need to discuss how we can run these tests in US.

File: salt/default/hostname.sls [hunk 1]
```diff
@@ -46,4 +46,17 @@ hosts_file_hack:
     {% else %}
       pythonexec: python
     {% endif %}
+{% else %}
+add_domain_to_searchlist:
```
Comment (moio) [inline salt/default/hostname.sls]: Why is this needed?

File: salt/default/repos.sls [hunk 2]
```diff
@@ -305,8 +287,8 @@ tools_additional_repo:
 
 tools_additional_repo:
   file.managed:
-    - name: /etc/zypp/repos.d/Devel_Galaxy_Manager_3.1_SLE-Manager-Tools-15-x86_64.repo
-    - source: salt://default/repos.d/Devel_Galaxy_Manager_3.1_SLE-Manager-Tools-15-x86_64.repo
+    - name: /etc/zypp/repos.d/Devel_Galaxy_Manager_3.2_SLE-Manager-Tools-15-x86_64.repo
```
Comment (moio) [inline salt/default/repos.sls]: Will this also work for `3.1-nightly` clients/mirrors?

File: salt/default/repos.sls [hunk 3]
```diff
@@ -395,27 +377,18 @@ tools_update_repo:
     - require:
       - cmd: galaxy_key
 
-{% elif '3.0-nightly' in grains.get('version') | default('', true) %}
-tools_update_repo:
-  file.managed:
-    - name: /etc/yum.repos.d/Devel_Galaxy_Manager_3.0_RES-Manager-Tools-7-x86_64.repo
```
Comment (moio) [inline salt/default/repos.sls]: It was my understanding that we could use `Devel:` client tools repos for any OS and any supported Manager client version - so we should have `Devel_Galaxy_Manager_3.{1,2,3}_{SLE-Manager-Tools-11,SLE-Manager-Tools-12,SLE-Manager-Tools-15,RES-Manager-Tools-7}`.

Does this make sense?

File: salt/default/hostname.sls (hunk 1 above)
Comment (mcalmer) [reply salt/default/hostname.sls]: I opened a PR to the SCC dhcp config which provide a searchlist.
I hope the approve this fast and then we do not need this anymore.

File: salt/default/repos.sls (hunk 2 above)
Comment (mcalmer) [reply salt/default/repos.sls]: We only use the "latest" for all SUSE Manager versions. And we are now about to switch to 3.2 versions of the repos.

I have removed the unused repo definitions from salt and minima (https://github.com/moio/sumaform/pull/357/commits/30901814a20f3bf3faf8e7a654b919adbe1f2685)

File: salt/default/repos.sls (hunk 3 above)
Comment (mcalmer) [reply salt/default/repos.sls]: Any OS yes, but not any SUMA version. All Clients use the same version independent of the Server version where they are connected to.

**Comment by mcalmer** [comment]
@moio - I think now we are ready to merge. Maybe you want to check again?
